app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -77,4 +78,43 @@ abstract class AppDatabase : RoomDatabase() {
         """)
     }
   }
+
+  class Migration_4_5 : Migration(4, 5) {
+
+    override fun migrate(database: SupportSQLiteDatabase) {
+      // Update local searchable name in the Patient table to strip out the newly added characters
+      database.inTransaction {
+        compileStatement("""update "Patient" set "searchableName"=? where "uuid"=?""")
+            .use { statement ->
+              query("""select "uuid","fullName" from "Patient"""")
+                  .use { cursor ->
+                    val uuidIndex = cursor.getColumnIndex("uuid")
+                    val fullNameIndex = cursor.getColumnIndex("fullName")
+
+                    generateSequence { if (cursor.moveToNext()) cursor else null }
+                        .map { it.getString(uuidIndex) to it.getString(fullNameIndex) }
+                        .map { (uuid, fullName) -> uuid to convertNameToSearchableForm(fullName) }
+                        .forEach { (uuid, searchableName) ->
+                          statement.bindString(1, searchableName)
+                          statement.bindString(2, uuid)
+                          statement.executeUpdateDelete()
+                        }
+                  }
+            }
+      }
+    }
+  }
 }
+
+/**
+ * Execute the given block in a transaction. Will call [SupportSQLiteDatabase.setTransactionSuccessful] only if no errors were thrown within the block
```

Comment:
Can you break this comment into multiple lines? There's no strict rule on this, but we try keeping a maximum of 13 words in a line for readability.

Reply:
Can you also end this comment with a period?

Reply:
Actually, I just read the code. I think this comment is documenting *what* the code is doing. Considering the code is very simple to understand, I don't think we need this comment here :)